src/main/kotlin/com/ampnet/projectservice/persistence/model/Organization.kt

```diff
@@ -37,11 +37,11 @@ class Organization(
 
     @OneToMany(fetch = FetchType.LAZY)
     @JoinColumn(name = "organizationUuid")
-    var documents: MutableList<Document>?,
+    var documents: MutableSet<Document>?,
```

Comment:
I'm not sure that replacing the list with a set optimises the performance of hibernate. https://thorben-janssen.com/association-mappings-bag-list-set/
According to the article  if you map your to-many association to a java.util.List without specifying the order of your association elements, you’re using a Bag which is like an unordered collection, and has slightly better performance than set.

Reply:
If it is defined as a list it cannot be fetched in the query.  "A List might be more efficient than a Set" is not enough to create an additional query for each project.

Reply:
For future info: https://vladmihalcea.com/hibernate-multiplebagfetchexception/
Problem is not in using set, but it enables you to fetch huge amounts of data. Use join fetch carefully.